loop-templates/templates/src/aptitudes/cursor/cursorListener.ts.squirrelly

```diff
@@ -0,0 +1,50 @@
+import { cursor, keyboard, whisper } from '@oliveai/ldk';
+import CursorWhisper from '../../whispers/CursorWhisper';
+
+export async function hotkeyCheck(char{{ it.isTypeScript ? ': string' : ''}} ) {
```

Comment:
I'd recommend not mixing const arrow functions and keyword functions. Doesn't really matter which you pick but it's better to be consistent